
/media/cal314-1/9E044F59044F3415/Marcin/noisy_labels/Dataset/SegSperm/test
GPU dostÄ™pne: NVIDIA GeForce RTX 3090
Traceback (most recent call last):
  File "/media/cal314-1/9E044F59044F3415/Marcin/noisy_labels/Kod/inference.py", line 438, in <module>
    predict(model, test_loader)
  File "/media/cal314-1/9E044F59044F3415/Marcin/noisy_labels/Kod/inference.py", line 243, in predict
    intersections_plot = intersections.transpose((0, 2, 3, 1))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: transpose() received an invalid combination of arguments - got (tuple), but expected one of:
 * (int dim0, int dim1)
 * (name dim0, name dim1)